api/rest/client/lbclient.go

```diff
@@ -44,20 +45,23 @@ func (r *RoundRobin) SetClients(cl []Client) {
 	r.length = uint32(len(cl))
 }
 
-// Failover is a load balancing strategy that would try the local cluster peer first.
-// If the local call fail it would try other client in a round robin fashion.
+// Failover is a load balancing strategy that would try the local cluster peer
```

Comment:
it's not the local, it's the first client. Whether that's a local peer or not is not the LB client problem...